webpack/scenes/Subscriptions/components/SubscriptionsTable/SubscriptionTypeFormatter.js

```diff
@@ -6,14 +6,21 @@ import { urlBuilder } from 'foremanReact/common/urlHelpers';
 export const subscriptionTypeFormatter = (value, { rowData }) => {
   let cellContent;
 
+  const clickHandler = (hypervisorLink) => {
+    window.location = hypervisorLink;
+  };
+
   if (rowData.virt_only === false) {
     cellContent = __('Physical');
   } else if (rowData.hypervisor) {
+    const hypervisorLink = urlBuilder('content_hosts', '', rowData.hypervisor.id);
     cellContent = (
       <span>
         {__('Guests of')}
         {' '}
-        <Link to={urlBuilder('content_hosts', '', rowData.hypervisor.id)}>{rowData.hypervisor.name}</Link>
+        <Link to={hypervisorLink} onClick={() => clickHandler(hypervisorLink)}>
+          {rowData.hypervisor.name}
+        </Link>
```

Comment:
For non-React pages, the React Router `<Link>` won't work out of the box, as you've discovered.

I would be partial to using a regular old-fashioned `<a>`:
```suggestion
        <a href={hypervisorLink}>{rowData.hypervisor.name}</a>
```

This removes the need for both the onClick and the `clickHandler` function.